render.yaml

```diff
@@ -0,0 +1,225 @@
+# Configure deployment on Render.com infrastructure.
+# For info on options, see: https://render.com/docs/blueprint-spec
+#
+# THIS IS AN EXPERIMENT. Our primary deployment is still on AWS, but we hope to
+# switch to this if it works well and is not too expensive. See:
+# https://github.com/usdigitalresponse/univaf/issues/683
+
+databases:
+  - name: univaf-db-production
+    plan: pro
+    databaseName: univaf_db_production
+    user: univaf
+    # Not sepecifying IPs for now; we may want to set this to an empty array
+    # to disallow public connections, though. i.e. `ipAllowList: []`
+    # ipAllowList:
+    #   - source: 203.0.113.4/30
+    #     description: office
+
+services:
+  - name: API Server
+    type: web
+    plan: standard
+    env: node
+    repo: "https://github.com/usdigitalresponse/univaf.git"
+    scaling:
+      minInstances: 1
+      maxInstances: 6
+      targetMemoryPercent: 60 # optional if targetCPUPercent is set
+      targetCPUPercent: 60 # optional if targetMemory is set
+    buildCommand: "cd server && npm install && npm run build"
+    startCommand: "cd server && npm run migrate && node ./dist/src/server.js"
+    healthCheckPath: "/health"
+    envVars:
+      - fromGroup: "Server Environment"
+
+  # Dump database and historical log files in S3
+  - name: "Daily Data Snapshot"
+    type: cron
+    schedule: "0 5 * * *"
+    plan: standard
+    env: node
+    repo: "https://github.com/usdigitalresponse/univaf.git"
+    branch: main
+    buildCommand: |
+      cd server
+      npm install && npm run build
+    startCommand: |
+      cd server
+      node scripts/availability_dump.js --write-to-s3 --clear-log
+    envVars:
+      - fromGroup: "Server Environment"
+      - key: DATA_SNAPSHOT_S3_BUCKET
+        sync: false
+      - key: AWS_ACCESS_KEY_ID
+        sync: false
+      - key: AWS_SECRET_ACCESS_KEY
+        sync: false
+      - key: AWS_DEFAULT_REGION
+        sync: false
+
+  # Loaders -----------------------------------------------------------------
+  # NOTE: This first loader also defines properties for re-use by the others.
+  - &loader-config
+    name: "Loader: CVS SMART"
+    schedule: "0/10 * * * *" # Every 10 minutes
+    startCommand: |
+      cd loader
+      bin/univaf-loader cvsSmart --send --compact --states "AL,AK,AZ,AR,CA,CO,CT,DE,DC,FL,GA,HI,ID,IL,IN,IA,KS,KY,LA,ME,MD,MA,MI,MN,MS,MO,MT,NE,NV,NH,NJ,NM,NY,NC,ND,OH,OK,OR,PA,RI,SC,SD,TN,TX,UT,VT,VA,WA,WV,WI,WY,MH,PR,VI"
+    # Common options used for all crons. (Other configs use `<<` to repeat this
+    # cron job's configuration and override the above, non-common values.)
+    plan: starter
```

Comment:
I went with a smaller plan for the loaders, since we are almost certainly way overprovisioned for them on AWS right now (see “resize our application's resource requests” in #303).